tests/providers/microsoft/azure/operators/test_azure_service_queue.py

```diff
@@ -0,0 +1,234 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import datetime
+import unittest
+from unittest import mock
+
+from azure.servicebus import ServiceBusMessage
+
+from airflow.models.dag import DAG
+from airflow.providers.microsoft.azure.operators.azure_service_bus_queue import (
+    AzureServiceBusCreateQueueOperator,
+    AzureServiceBusDeleteQueueOperator,
+    AzureServiceBusReceiveMessageOperator,
+    AzureServiceBusSendMessageOperator,
+)
+
+QUEUE_NAME = "test_queue"
+OWNER_NAME = "airflow"
+DAG_ID = "test_azure_service_bus_queue"
+
+
+class TestAzureServiceBusCreateQueueOperator(unittest.TestCase):
+    def setUp(self):
+        args = {'owner': OWNER_NAME, 'start_date': datetime.datetime(2017, 1, 1)}
```

Comment:
nit: How about removing dag param since the intention here is to test the task only if you do so you do not need the setUp method also